src/rpc/RPCHelpers.cpp

```diff
@@ -220,6 +218,11 @@ getAccount(
         return {};
     }
 
+    if (!request.contains(account))
```

Comment:
This line doesn't build. Also, this is not actually what we want to do. We want to check if the account exists on the ledger.